backend/src/decorators.py

```
@@ -0,0 +1,20 @@
+from functools import wraps
+from flask import jsonify
+from flask_jwt_extended import verify_jwt_in_request, get_jwt_identity
+from src.models.database import User
+
+def admin_required(fn):
```

Ruff (0.11.9)

Missing return type annotation for public function `admin_required`

(ANN201)